src/tab/Tab.tsx

```diff
@@ -49,6 +49,13 @@ function Tab({
   const [activeTabIndex, setActiveTabIndex] = useState(initialActiveTabIndex);
   const tabClassName = classNames("tab", customClassName);
 
+  let activeTabIndexToUse = activeTabIndexFromProps === undefined
+    ? activeTabIndex
+    : activeTabIndexFromProps;
+
+  // check if active index is out of bounds
+  activeTabIndexToUse = activeTabIndexToUse > items.length ? 0 : activeTabIndexToUse;
```

Comment:
```suggestion
  const [activeTabIndex, setActiveTabIndex] = useState(initialActiveTabIndex);

  // If activeTabIndex is uncontrolled, we need to use the local state
  let activeTabIndexToUse =
    activeTabIndexFromProps === undefined ? activeTabIndex : activeTabIndexFromProps;

  /**
   * In case activeTabIndex is out of bounds, we need to reset it to 0.
   * This can happen if items are changes after the mount
   */
  if (activeTabIndexToUse > items.length) {
    activeTabIndexToUse = 0;
  }

  const tabClassName = classNames("tab", customClassName);
```

What do you think about this version?
- I think it's better if we explain the things here better, it can be confusing since we have 3 variables for the same value (activeTabIndex, activeTabIndexFromProps, activeTabIndexToUse)
- `a = condition ? 0 : a` actually includes a single case (set 0 if `condition`), so I believe no need for ternary(?) here and an if block can make it more readable

Reply:
I agree, it's way more readable. Thanks 🙏